src/backend/src/services/projects.services.ts

```diff
@@ -602,4 +602,23 @@ export default class ProjectsService {
       })
     ).map(linkTypeTransformer);
   }
+
+  /**
+   * Create a new material type
+   * @param name the name of the new material type
+   * @param submitter the user who is creating the material type
+   */
+  static async createMaterialType(name: string, submitter: User): Promise<Material_Type> {
+    if (!isLeadership(submitter.role)) throw new AccessDeniedAdminOnlyException('create material type');
+
+    const materialType = await prisma.material_Type.create({
+      data: {
+        name,
```

Comment:
name is a unique id for materialType, so you need to check that a materialType with that name doesn't exist (and throw an error if it does) before trying to create the material type